src/actions/public/simulateDepositETH.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Transport,
+  PublicClient,
+  Chain,
+  SimulateContractParameters,
+  SimulateContractReturnType,
+} from 'viem'
+import { l1StandardBridgeABI } from '@eth-optimism/contracts-ts'
+import { OpChainL2 } from '@roninjin10/rollup-chains'
+import { simulateContract } from 'viem/actions'
+import { DepositETHParameters } from '../../types/depositETHParameters'
+
+export type SimulateDepositETHParameters<
+  TChain extends Chain | undefined = Chain,
+  TChainOverride extends Chain | undefined = Chain | undefined,
+> = Omit<
+  SimulateContractParameters<
+    typeof l1StandardBridgeABI,
+    'depositETH',
+    TChain,
+    TChainOverride
+  >,
+  'abi' | 'functionName' | 'args' | 'address'
+> & {
+  toChain: OpChainL2
+  args: DepositETHParameters
+}
+
+export type SimulateDepositETHReturnType<
```

Comment:
Do we need this 🤔 ? Can we not just use `SimulateContractReturnType`

Reply:
It makes the typing in publicOpStack ugly to not have this